src/dependecies.py

```diff
@@ -0,0 +1,7 @@
+from cache import RedisCacheManager
+
+cache_manager = RedisCacheManager()
+
+
+def get_cache_manager() -> RedisCacheManager:
```

Comment:
We can probably cache that using lru cache for instance

Reply:
Don't know what is it, but will try

Reply:
probably it makes no sense here since you are instanciating a redis manager outside the function

Reply:
But I still would like to learn about lru cache, so will check it)